stored_requests/config/config.go

```diff
@@ -84,12 +84,12 @@ func newFetchers(cfg *config.StoredRequests, client *http.Client, db *sql.DB) (f
 		idList = append(idList, fFetcher)
 		ampIDList = append(ampIDList, fFetcher)
 	}
-	if cfg.Postgres != nil {
+	if cfg.Postgres.FetcherQueries.QueryTemplate != "" {
```

Comment:
Do you think we should split this into two clauses now? Someone might define `QueryTemplate` and not `AmpQueryTemplate`, or vice versa.

Otherwise we should add some `validate()` logic to make sure that both or neither are defined... but I lean towards changing the logic here so that it works either way.

Reply:
I think our logic breaks in other places if someone defines one and not the other. Before if the struct was not null, we just assumed we had both and configured the fetchers appropriately. We will likely have to dive deeper into our code if we want to support piecemeal configs.

Reply:
if it doesn't work today, separate PR.

Reply:
Fixing the base issues was enough work for this PR. Adding support for partial configs (something we don't support today) left for another PR.